schema/src/project.yaml

```diff
@@ -15,23 +15,42 @@ default_range: string
 
 classes:
   Project:
+    tree_root: true
```

Comment:
Is it important to set `tree_root` here? Presumably this helped to create the `project-data.*` files?

Just worry because it says only one class can be set as a root. We [import](https://linkml.io/linkml/schemas/imports.html) classes from multiple schemas which might cause an issue.

From https://linkml.io/linkml/intro/tutorial02.html:

> In LinkML, the tree_root slot is used to designate a class as the root of a tree structure. Only one class in a schema can be set as root. If more than one class is marked as tree_root, a validation error will occur.

> Marking one class to serve as the root of the tree (as “container” of the other classes) is especially important when serializing and deserializing data. The class marked as tree_root will be the top-level object in the serialized data.

Reply:
removed